### PR TITLE
[RHPAM-4404] KieApp missing description for terminationRoute property

### PR DESCRIPTION
**Pull request by desmax74**
Signed-off-by: desmax74 <mdessi@redhat.com>
See:https://issues.redhat.com/browse/RHPAM-4404

### CODE REVIEW

**Review by davidesalerno — APPROVED** [review]

File: pkg/apis/app/v2/kieapp_types.go [hunk 1]
```diff
@@ -160,6 +160,7 @@ type KieServerSet struct {
 	TerminationRoute         *TerminationRoute `json:"terminationRoute,omitempty"`
 }
 
+// TerminationRoute configuration for the routes in the Workbench, KieServer, Dashbuilder,SmartRouter, Pim
```
Comment (spolti) [inline pkg/apis/app/v2/kieapp_types.go:163]:
```suggestion
// TerminationRoute configuration of the routes used by RHPAM components
```

**Review by spolti — CHANGES_REQUESTED** [review]

**Comment by desmax74** [comment]
@spolti on the main https://github.com/kiegroup/kie-cloud-operator/pull/798

**Pull request merged**